File: bee_1192.c
#include <stdio.h>

int main() {
    int numCasos;
    scanf("%d", &numCasos);
    00
    for (int i = 0; i < numCasos; i++) {
        char entrada[4];
        scanf("%s", entrada);
        
        int digito1 = entrada[0] - '0';
        int digito2 = entrada[2] - '0';
        
        if (digito1 == digito2) {
            printf("%d\n", digito1 * digito2);
        } else if (entrada[1] >= 'A' && entrada[1] <= 'Z') {
            printf("%d\n", digito2 - digito1);
        } else {
            printf("%d\n", digito2 + digito1);
        }
    }
    
    return 0;
}

File: bee_1192.py
def main():
    numCasos = int(input())
    for _ in range(numCasos):
        entrada = input()
        digito1 = int(entrada[0])
        digito2 = int(entrada[2])
        if digito1 == digito2:
            print(digito1 * digito2)
        elif 'A' <= entrada[1] <= 'Z':
            print(digito2 - digito1)
        else:
            print(digito2 + digito1)
    pass

main()

